src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -793,7 +793,13 @@ export class RoslynLanguageServer {
         vscode.commands.registerCommand(
             DynamicFileInfoHandler.dynamicFileUpdatedCommand,
             async (notification: RazorDynamicFileChangedParams) =>
-                this.sendNotification<RazorDynamicFileChangedParams>('razor/dynamicFileInfoChanged', notification)
+            {
+                if (this.isRunning()) {
+                    this.sendNotification<RazorDynamicFileChangedParams>('razor/dynamicFileInfoChanged', notification)
+                } else {
+                    _channel.debug('Tried to send razor/dynamicFileInfoChanged while server is not running');
```

Comment:
If this is rare and only during shutdown, would it be better to log it as a more important message, so that if it appears in a user report of general instability, we'll know something is up with server lifetimes?